frontend/src/routes/+layout.ts

```diff
@@ -1,11 +1,9 @@
-import { browser } from '$app/environment';
 import '$lib/i18n'; // Import to initialize. Important :)
 import { locale, waitLocale } from 'svelte-i18n';
 import type { LayoutLoad } from './$types';
+import { localeStoragevalue } from '$lib/i18n/index.js';
 
 export const load: LayoutLoad = async () => {
-  if (browser) {
-    locale.set(window.navigator.language);
-  }
+  locale.set(localeStoragevalue());
```

Comment:
Don't we need to pass check here for `browser` with `if` or smth, to ensure, that everything work correctly?